collector/src/bin/rustc-fake.rs

```diff
@@ -71,6 +75,10 @@ fn print_memory() {
     }
 }
 
+fn print_time(dur: Duration) {
+    println!("{}.{:09};;wall-time;4;100.00", dur.as_secs(), dur.subsec_nanos());
```

Comment:
Could we add some documentation here? Especially around what the `4` represents. A link (since I assume this is the standard perf output) to documentation somewhere else would also be fine.